tools/lib/test_server.py

```diff
@@ -67,7 +67,7 @@ def test_server_running(force: bool=False, external_host: str='testserver',
         update_test_databases_if_required(rebuild_test_database=True)
 
     # Run this not through the shell, so that we have the actual PID.
-    run_dev_server_command = ['tools/run-dev.py', '--test']
+    run_dev_server_command = ['tools/run-dev.py', '--test', '--streamlined']
```

Comment:
I don't see any perf benefit to this option when testing locally, but I have 6 cores.  Do you?  It'd be nice to have a note on perf impact for this in the commit message.

Reply:
I'm on a pretty pristine droplet...I think the likely folks to be impacted here are folks running on Vagrant on a slow laptop.  I'll try to do a quick measurement, though.